### DISCUSSION TITLE
SCRUM PREP MILESTONE 2-2

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#186] [General]
This week i met up with the manager to discuss total points for this milestone. Started researching on the lecture topic tasks and held scrum meetings with the team leads.

**Comment by sandy-marrero** [comment]
- Started working on issue #179.
- PR for #131 was merged.

**Comment by JadiellMV** [comment]
This week I added error handling whenever a user tries to sign in an existing username or email and when in the login, the password doesn't match the username/email. Started working on a password reset feature.

**Reply by PinkSylvie** [reply]
This week as soon as the documentation grade was received I created the tasks for teammates to fix the issues that appeared in the milestone 1 documentation to avoid those tasks being affected by the decay.

**Comment by AnthonyMed13** [comment]
Finished working on issue #120 which was to implement factories

**Comment by ianramospr** [comment]
Started working on issue #177.

**Comment by DavidCuevas3** [comment]
Continued brainstorming on possible changes to the multiplayer leaderboard before working on implementing it completely. Issue: #171

**Comment by Saul-Caban-UPRM** [comment]
Started to work on task #157, encountered some road blocks, but hopefully I’m able to overcome them soon

**Comment by Adriana-Isabel** [comment]
I am working on creating a Home Page that includes the header, body and the footer. Since I am working on three tasks at the same time, I believe it could be ready for next week, if everything goes without problems.

**Comment by ebrain-mendoza121** [comment]
Attended team meetings, worked on expanding lobby functionality to later handle different setting selection at the moment of creation.

**Comment by Joaquin939** [comment]
This week i was able to create 2 new issues

**Comment by AngelCIICMorales** [comment]
Organized meetings, reviewed pull requests, and spoke to team leaders.

**Comment by ilikecheese117** [comment]
Created lecture topic task and continued working on save system

**Comment by Negroni1** [comment]
Worked on domain model diagram and layered architecture diagram

**Comment by RicardoJRSs** [comment]
Worked on lobby design functionality and documentation

**Comment by FrancesC8** [comment]
Deployed the frontend app and connected it to the latest deployed version of the backend here: https://multi-dominoes-frontend.vercel.app/ , [PR](https://github.com/uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes/pulls/207) is ready for review.

**Comment by Jvstinoo** [comment]
Implemented #156 to create a placeholder screen for choosing the game mode. Created PR: #208.

**Comment by fernandopizarro1** [comment]
Finished the IntermediateBot #151 and also have continued working on the AdvancedBot #158. Also attended meetings to discuss different implementations.

**Comment by rukitoyordan** [comment]
Worked on providing an LTT and planning for new designs on powerups.

**Comment by Gaby-Concepcion** [comment]
Made progress on adding the new drawing game mode to the rules engine, and additionally lined up a few issues relating to touching up and expanding on the documentation. Specifically expanding and reworking the User stories, Epics, and Features, Adding a Narrative, and adding Factories.

**Comment by G4vp** [comment]
Started to work with the issue #159.

**Comment by print-Eruki** [comment]
Unit testing para generation of UUID and integration testing with backend (in progress

**Comment by frances-sepulveda** [comment]
This week I worked with the palettes and designs to make sure they are compatible for the game.

**Comment by nsfrontera** [comment]
Working on completing my task, have planned on finishing one task for this sprint.
